src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1769,7 +1781,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     // - wstr: the string of characters to write to the terminal connection.
     // Return Value:
     // - <none>
-    void TermControl::_SendInputToConnection(const std::wstring& wstr)
+    void TermControl::_SendInputToConnection(const winrt::param::hstring& wstr)

```

Comment:
unfortunately ☹️ we can't actually use the `winrt::param` namespace _even though it is literally the best namespace for taking in anything that can convert to hstring_

We might have to prefer `std::wstring_view`, which can be natively converted to an hstring (though it could also crash because hstring expects string_views to be null-terminated)

Reply:
(I've discussed this with the cppwinrt folks before. param::hstring does everything we might possibly want with regards to automatic promotion to hstring, but it's not part of the public interface :/)

Reply:
I'm not entirely sure I understand... Am I supposed to not use `winrt::param` because it's a private interface, or is there another technical reason?

Reply:
Oh, just because it is a private namespace.